rancid-tomatillos/src/Components/Header/Header.js

```diff
@@ -1,16 +1,23 @@
 import React from 'react';
 import './Header.scss';
+import { connect } from 'react-redux';
 
-const Header = () => {
+
+const Header = ({ data }) => {
+  const index = Math.floor(Math.random() * Math.floor(20));
   return (
-    <header className='header-main'>
+    <header className='header-main' style={{backgroundImage: `url(${data.movies[index].backdrop_path})`}}>
       <h1 className='header-h1'>Rancid Tomatillos</h1>
       <section className='header-btn-section'>
-        <button className='header-login-btn'>Log In</button>
-        <button className='header-home-btn'>Home</button>
+        <button className='header-btn'>Log In</button>
```

Comment:
At some point we will want to dynamically update the inner html of this button, and so we should give it an id to be able to grab it